frontend/src/util/auth.tsx

```diff
@@ -15,6 +17,9 @@ interface User {
 
 const fakeFirebaseUser: User = {
```

Comment:
I'm using fakeFirebaseUser for PasswordAuth mode as well with minor modification
I'm still considering a seperate object to be used specifically for PasswordAuth.

That would probably have been cleaner but verbose